components/SearchList.js

```diff
@@ -0,0 +1,49 @@
+import styled from "styled-components";
+
+export default function SearchList({ result, handleClick }) {
+  return (
+    <>
```

Comment:
You can delete this fragment because `StyledSearchListWrapper` is the only parent element in this return